sh/test.sh

```diff
@@ -27,9 +27,9 @@ startfold "Running Integration Tests..." "test-integration";
 endfold "test-integration" ;
 
 #run server-unit test
-startfold "Running server Unit Tests ......" "server-unit"
-./node_modules/.bin/mocha --recursive --exit test/server-unit
-endfold "server-unit" ;
+# startfold "Running server Unit Tests ......" "server-unit"
+# ./node_modules/.bin/mocha --recursive --exit test/server-unit
+# endfold "server-unit" ;
```

Comment:
Is it possible to add a comment about the fact that server-unit tests are running on `appveyor` or just remove these commented codes.

Reply:
Good point.  I'll remove this code.